aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/input/InputStepTest.java

```diff
@@ -64,5 +97,10 @@ public void parameter() throws Exception {
         // make sure 'x' gets assigned to false
         System.out.println(b.getLog());
         assertTrue(b.getLog().contains("after: false"));
+
+        //make sure the approver name corresponds to the submitter
+        ApproverAction action = b.getAction(ApproverAction.class);
+        assertNotNull(action);
+        assertEquals(is.getInput().getSubmitter(), action.getUserId());
```

Comment:
`getUserId` is fine here.

You could simplify slightly to

```java
assertEquals("alice", action.getUserId());
```

but it does not really matter.